Doc/library/zipfile.rst

```diff
@@ -637,22 +641,41 @@ file:
    .. versionadded:: 3.8
       The *strict_timestamps* keyword-only argument
 
+   .. versionadded:: 3.9
+      The *follow_symlinks* keyword-only argument
+
 
 Instances have the following methods and attributes:
 
 .. method:: ZipInfo.is_dir()
 
-   Return ``True`` if this archive member is a directory.
+   Return :const:`True` if this archive member is a directory.
 
+   .. versionadded:: 3.6
    This uses the entry's name: directories should always end with ``/``.
 
-   .. versionadded:: 3.6
+   .. versionchanged:: 3.9
+   Use the filemode contained in external file attributes.
+
+.. method:: ZipInfo.is_reg()
 
+   Return :const:`True` if this archive member is a regular file.
+
+.. method:: ZipInfo.is_file()
+
+   Same as :meth:`is_reg`.
+
+.. method:: ZipInfo.is_sym()
+
+   Return :const:`True` if this archive member is a symbolic link.
 
```

Comment:
I think you removed this.
```suggestion

.. versionadded:: 3.6
```

Reply:
This was for `ZipInfo.is_dir()` and still is, I just moved it one line up to underline the way this method is coded changes with this PR. Maybe I got the syntax or the conventions wrong however, I'm not used to reStructuredText.